starter-code/iteration-1/main.ts

```diff
@@ -1,12 +1,26 @@
 // Iteration 1
-// 1. Create an array of strings
-
-// 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
-// 3. Create a function to list all tasks, it must show in the console de task.
-
-// 4. Create a function to delete a task, you must find the task inside the array and delete it.
-
+  // 1. Create an array of strings
+  let arrayStrings: Array<string> = [];
+  
+  // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+ let task:string;
+  
+ function addTask(task):number{
```

Comment:
You put the type here, with the parameter, not defining a global variable as you have done in line 6:
```
function addTask(task: string): number {
```